src/app/src/util/util.js

```diff
@@ -52,7 +55,18 @@ export const makeGetFacilitiesURL = () => '/api/facilities/';
 export const makeGetFacilityByOARIdURL = oarId => `/api/facilities/${oarId}/`;
 export const makeGetFacilitiesURLWithQueryString = qs => `/api/facilities/?${qs}`;
 
-export const getValueFromObject = ({ value }) => value;
+
+// Get a value from a one of the React Select filter option objects if it also
+// has a label. (Values without labels were hydrated from the query-string but
+// not subsequently confirmed by the API-sent list of available options, so we
+// discard them.
+export const maybeGetValueFromFilterObject = ({ value, label = null }) => {
```

Comment:
I think this change can be dropped, as it was from an iteration of this where I was trying to distinguish selections staged from the querystring params and those validated by the choices returned from the API by not providing a label for the staged ones. I updated it later to use the value as the label until the selection had been validated, then replace it.